sdk/redis/Azure.ResourceManager.Redis/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.0.2 (2022-10-01)
```

Comment:
Please also update the version in Azure.ResourceManager.Redis.csproj

Reply:
I think I did, no? https://github.com/Azure/azure-sdk-for-net/pull/31571/files#diff-5ebfe85f03892eb70b80815cdd47c2266ddddd366d44786fb5bc6e2bf48f2667